mail_broker_whatsapp/models/mail_broker_whatsapp.py

```diff
@@ -0,0 +1,361 @@
+            if related_message_id:
+                related_message = (
+                    self.env["mail.notification"]
+                    .search(
+                        [
+                            ("broker_channel_id", "=", chat.id),
+                            ("broker_message_id", "=", related_message_id),
+                        ]
+                    )
+                    .mail_message_id
+                )
+                if related_message and related_message.broker_message_id:
+                    new_related_message = (
+                        self.env[related_message.broker_message_id.model]
+                        .browse(related_message.broker_message_id.res_id)
+                        .message_post(
+                            body=body,
+                            author_id=author
+                            and author._name == "res.partner"
+                            and author.id,
+                            broker_type="whatsapp",
+                            date=datetime.fromtimestamp(int(message["timestamp"])),
+                            # message_id=update.message.message_id,
+                            subtype_xmlid="mail.mt_comment",
+                            message_type="comment",
+                            attachments=attachments,
+                        )
+                    )
+                    new_message.broker_message_id = new_related_message
+
+    def _send(
+        self, broker, record, auto_commit=False, raise_exception=False, parse_mode=False
+    ):
+        message = False
+        try:
+            attachment_mimetype_map = self._get_whatsapp_mimetype_kind()
+            for attachment in record.mail_message_id.attachment_ids:
+                if attachment.mimetype not in attachment_mimetype_map:
+                    raise UserError(_("Mimetype is not valid"))
+                attachment_type = attachment_mimetype_map[attachment.mimetype]
+                m = requests_toolbelt.multipart.encoder.MultipartEncoder(
+                    fields={
+                        "file": (
+                            attachment.name,
+                            base64.b64decode(attachment.datas),
+                            attachment.mimetype,
+                        ),
+                        "messaging_product": "whatsapp",
+                        # "type": attachment_type
+                    },
+                )
+
+                response = requests.post(
+                    "https://graph.facebook.com/v%s/%s/media"
+                    % (
+                        broker.whatsapp_version,
+                        broker.whatsapp_from_phone,
+                    ),
+                    headers={
+                        "Authorization": "Bearer %s" % broker.token,
+                        "content-type": m.content_type,
+                    },
+                    data=m,
+                    timeout=10,
+                )
```

Comment:
graph.facebook.com is banned in Russia. Could you please add proxy variable here?
Doc: https://github.com/psf/requests/blob/96b22fa18c00831656ee4b286bf1c9062459b00a/docs/user/advanced.rst#proxies
Noting that global proxy configuration via env can be not suiatable in all situations.